src/clarity-angular/dropdown/dropdown-submenu.ts

```diff
@@ -0,0 +1,41 @@
+import {Component, ContentChildren, Input, QueryList} from "@angular/core";
+import {DropdownItem} from "./dropdown-item";
+
+@Component({
+    selector: "clr-dropdown-submenu",
+    template: `
+        <ng-content></ng-content>
+    `,
+    host: {
+        "[class.dropdown-submenu]" : "true",
+        "[class.open]" : "open",
+        "[class.right]" : "openDirection === 'right'",
+        "[class.left]" : "openDirection === 'left'",
+        "(click)": "open = true",
```

Comment:
this is not an issue but just a question. angular recommends that we use `HostListener`s instead of this but I am not sure what convention we are following. I know we do regularly for classes but very rarely for events, no?